.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,9 +19,11 @@
 
 ### Pull request checklist
 
-This PR has considered:
-- [ ] Light and Dark appearances
-- [ ] VoiceOver and Keyboard Accessibility
-- [ ] Internationalization and Right to Left layouts
+This PR has considered (when applicable):
+- [ ] Unit tests
+- [ ] UI tests
 - [ ] Different resolutions (1x, 2x, 3x)
```

Comment:
Similar comment to above - still platform specific?

Reply:
I guess windows has a lot of snapshot tests which are cross plat. Sort of assumed they have something analogous to unit tests on other platforms but maybe they're named so differently that this doesn't make sense...